### PR TITLE
Feature/lga 2459 chs change in family scope

### PR DESCRIPTION
**Pull request by MishNajam**
## What does this pull request do?

Required.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [ ] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: behave/features/steps/frontend_assignment.py [hunk 1]
```diff
@@ -515,3 +528,59 @@ def step_impl_assign_f2f(context):
...
+@step("I select the parent option")
+def step_impl_select_family_option(context):
+    context.helperfunc.click_button(
+        By.CSS_SELECTOR, "input[type='radio'][value='n411']"
+    )
+    print("clicked")
+    context.helperfunc.click_button(By.NAME, "diagnosis-next")
+    print("selected next")
+
+
```
Comment (MishNajam) [inline behave/features/steps/frontend_assignment.py:552]: merge these two functions, remove print statements comments

File: behave/features/steps/frontend_assignment.py [hunk 2]
```diff
@@ -515,3 +528,59 @@ def step_impl_assign_f2f(context):
...
+def step_impl_select_other_option(context):
+    context.helperfunc.click_button(
+        By.CSS_SELECTOR, "input[type='radio'][value='n412']"
+    )
+    print("clicked")
+    context.helperfunc.click_button(By.NAME, "diagnosis-next")
+    print("selected next")
+
+
+@step("I select the diagnosis Family and click next once")
```
Comment (MishNajam) [inline behave/features/steps/frontend_assignment.py:563]: refactor function from copied one

**Review by psweeting1 — APPROVED** [review]
LGTM nice job

**Review by Pheonnexx — APPROVED** [review]
LGTM!

**Pull request merged**